src/logic/operate.js

```diff
@@ -0,0 +1,21 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const num1 = Big(numberOne);
+  const num2 = Big(numberTwo);
+  let result = 0;
+
+  const operations = {
+    '+': () => num1.plus(num2),
+    '-': () => num1.minus(num2),
+    x: () => num1.times(num2),
+    '÷': () => num1.div(num2),
+    '%': () => num1.mod(num2),
```

Comment:
The mod operation in here actually transforms the number into a percentage based number. For example,
the following sequence(80 % 20), will result in '0.820'
How? Ans: 80 get divided by 100 and 20 get concatenated with the calculated result.